# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/SocketProcessChild.cpp
# Commit: 6e1eba5c88ad
# Full Hash: 6e1eba5c88ad47f0a2a3aa37b4c2cae9f31c2ef3
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 1
# Description:
#   Bug 1569183: Stop doing a proxy lookup to determine whether we're configured to use a proxy (for the proxy_only_if_behind_proxy pref), and instead look at whether we loaded the doc using a proxy. r=mjf,mayhemer,jld
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45289
# ==============================================================================

diff -r c43589fd0f90 -r 6e1eba5c88ad netwerk/ipc/SocketProcessChild.cpp
--- a/netwerk/ipc/SocketProcessChild.cpp	Tue Sep 17 18:18:33 2019 +0000
+++ b/netwerk/ipc/SocketProcessChild.cpp	Wed Sep 18 14:27:42 2019 +0000
@@ -23,7 +23,6 @@
 #endif
 
 #ifdef MOZ_WEBRTC
-#  include "mozilla/net/ProxyConfigLookupChild.h"
 #  include "mozilla/net/WebrtcTCPSocketChild.h"
 #endif
 
@@ -228,18 +227,5 @@
   return true;
 }
 
-PProxyConfigLookupChild* SocketProcessChild::AllocPProxyConfigLookupChild() {
-  MOZ_CRASH("AllocPProxyConfigLookupChild should not be called");
-  return nullptr;
-}
-
-bool SocketProcessChild::DeallocPProxyConfigLookupChild(
-    PProxyConfigLookupChild* aActor) {
-#ifdef MOZ_WEBRTC
-  delete static_cast<ProxyConfigLookupChild*>(aActor);
-#endif
-  return true;
-}
-
 }  // namespace net
 }  // namespace mozilla